reverie/backend_server/persona/prompt_template/run_gpt_prompt.py

```diff
@@ -1371,6 +1403,12 @@ def get_fail_safe():
   return output, [output, prompt, gpt_param, prompt_input, fail_safe]
 
 
+class DecideToReact(BaseModel):
+  '''
+  Should be a decision 1,2, or 3
+  '''
+  decision: int
+
```

Comment:
When there's only a small set of specific values that are valid, you can use an Enum (short for enumerated) to constrain the model field: https://docs.pydantic.dev/2.0/usage/types/enums/

Something like this might work:
```
class DecideToReactEnum(IntEnum):
  one = 1
  two = 2
  three = 3
  
class DecideToReact(BaseModel):
  decision: DecideToReactEnum
```

Reply:
Ahh got it